dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml

```diff
@@ -90,23 +87,77 @@
+				<not><isTrue value="${fat.test.databases}" /></not>
+			</and>
+		</condition>
+		
+		<!-- If a specific DB is being used, include it later in the bucket name -->
+    	<condition property="suite.db.type" value=" with ${fat.bucket.db.type}" else="">
+    		<and>
+    			<istrue value="${fat.test.databases}" />
+    			<isset property="fat.bucket.db.type" />
+    		</and>
+    	</condition>
 		            
-        <!-- If this test is compiled with a java level higher than the one we're currently running, -->
-        <!-- bypass the tests by invoking the always passes test instead of real FAT tests -->
-        <property file="${basedir}/fat.bnd.properties" />
+        <!-- Enforce fat.minimum.java.level rule. If this test is compiled with a java level 
+           higher than the one we're currently running we should skip these tests. -->
 		<script language="javascript"> <![CDATA[
-			var testBucketClassForLowJava = "componenttest.custom.junit.runner.AlwaysPassesTest";
 			var minTestJava = project.getProperty("fat.minimum.java.level");
 			var javaVersion = parseFloat(project.getProperty("java.specification.version"));
 			if(minTestJava != null && javaVersion < parseFloat(minTestJava))
-			  project.setProperty("test.bucket.class", testBucketClassForLowJava);
+				project.setProperty("skip.java.level", true);
         ]]></script>
+		
+		<!-- Output java level info -->
         <echo message="fat.minimum.java.level=${fat.minimum.java.level}"/>
         <echo message="java.specification.version=${java.specification.version}"/>
-        <echo message="FAT entry point will be: ${test.bucket.class}"/>
 		<echo message="micro.version=${micro.version}"/>
+		
+		<!-- Determine if we are going to skip testing or not -->
+		<!-- Bypass the tests by invoking the always passes test instead of real FAT tests -->
+		
+		<!-- Our build infrastructure will set this properties prior to launching tests, 
+			need to overwrite it here if we are going to skip tests. -->
+		<var name="test.bucket.class" unset="true" />
+		<iff>
+			<or>
+				<istrue value="${skip.no.db.tests}" />
+				<istrue value="${skip.java.level}" />
+			</or>
+			<then>
+		        <property name="test.bucket.class" value="componenttest.custom.junit.runner.AlwaysPassesTest" />
+				
+				<echo message="FAT bucket WILL NOT run tests, due to the following properties:"/>
+				<echo message="     Only run database rotation buckets:  ${fat.test.databases.only}"/>
+				<echo message="     This bucket uses database rotation:  ${fat.test.databases}"/>
+				<echo message="     This bucket uses java level:         ${fat.minimum.java.level}" />
+			</then>
+			<else>
+				<var name="test.bucket.name" unset="true" />
+		        <property name="test.bucket.class" value="${entry.point}" />
+				<property name="test.bucket.name" value="${test.bucket.class}${suite.db.type}"/>
+				
+				<echo message="FAT bucket WILL run, with the following properties:"/>
```

Comment:
Will these lines print out every time someone runs a FAT locally now? If so, I don't think it's necessary to log this much information on the "normal" path